src/styles/blocks/card.scss

```
@@ -0,0 +1,121 @@
+.card {
+  flex-direction: column;
+  display: flex;
+
+  transition: transform 0.3s;
+
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  background: $main-background-color;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+    font-size: $main-size;
+    line-height: $main-line-height;
+    font-weight: 500;
+    color: $main-accent;
+  }
+
+    &:hover &__name {
+      color: #34568b;
```

create variables for all colors
```suggestion
      color: ...;
```